css-three-column-layout/styles.css

```diff
@@ -0,0 +1,48 @@
+.container {
+  background-color: rgb(25 150 218);
+  color: white;
+  font-style: normal;
+  font-weight: 400;
+  margin: 0;
+}
+
+h1 {
+  display: flex;
+  justify-content: center;
+}
+
+.column-one-third {
+  display: inline-block;
+  flex-direction: row;
+  width: 33%;
+  text-align: center;
+  justify-content: center;
+}
+
+p {
+  width: 58%;
+  margin-left: 21%;
+  text-align: center;
+  justify-content: center;
```

Comment:
Not doing anything, beause there's no `display: flex`. `justify-content` is a parent property, which means you assign it to parent elements that are using `display: flex`